test/integration/smart_answer_flows/marriage_abroad_test.rb

```diff
@@ -802,8 +802,6 @@ def self.translations
   context "ceremony in thailand, resident in the UK, partner other" do
```

Comment:
Really minor thing, but I would change the title of this test to reflect that we are only testing for "opposite sex" now and not residence, or partner's nationality.